docs/src/main/docs/tracing/01_tracing.adoc

```diff
@@ -120,35 +120,171 @@ To create a custom span that is a child of the WebServer request:
+|`security`     |`security:outbound`    |Processing of outbound security
+|`jax-rs`       |A generated name       |Span for the resource method invocation, name is generated from class and method name
+|`jax-rs`       |`jersey-client-call`   |Span for outbound client call
+|===
+
+Some of these spans `log` to the span. These log events can be (in most cases) configured:
+
+|===
+|span name          |log name               |configurable   |enabled by default |description
+
+|`HTTP Request`     |`handler.class`        |YES            |YES                |Each handler has its class and event logged
+|`security`         |`status`               |YES            |YES                |Logs either "status: PROCEED" or "status: DENY"
+|`security:atn`     |`security.user`        |YES            |NO                 |The username of the user if logged in
+|`security:atn`     |`security.service`     |YES            |NO                 |The name of the service if logged in
+|`security:atn`     |`status`               |YES            |YES                |Logs the status of security response (such as `SUCCESS`)
+|`security:atz`     |`status`               |YES            |YES                |Logs the status of security response (such as `SUCCESS`)
+|`security:outbound`|`status`               |YES            |YES                |Logs the status of security response (such as `SUCCESS`)
+|===
+
+There are also tags that are set by Helidon components. These are not configurable.
+
+|===
+|span name          |tag name           |description
+
+|`HTTP Request`         |`component`        |name of the component - `helidon-webserver`, or `jaxrs` when using MP
+|`HTTP Request`         |`http.method`      |HTTP method of the request, such as `GET`, `POST`
+|`HTTP Request`         |`http.status_code` |HTTP status code of the response
+|`HTTP Request`         |`http.url`         |The path of the request (for SE without protocol, host and port)
+|`HTTP Request`         |`error`            |If the request ends in error, this tag is set to `true`, usually accompanied by logs with details
+|`content-read`         |`requested.type`   |Type (class) of the requested entity (if entity is read)
+|`content-write`        |`response.type`    |Type (class) of the entity being sent (if enitty is sent)
+|`security`             |`security.id`      |ID of the security context created for this request (if security is used)
+|`jersey-client-call`   |`http.method`      |HTTP method of the client request
+|`jersey-client-call`   |`http.status_code` |HTTP status code of client response
+|`jersey-client-call`   |`http.url`         |Full URL of the request (such as `http://localhost:8080/greet`)
+|===
+
+=== Traced spans configuration
+
+Each component and its spans can be configured using Config.
+The traced configuration has the following layers:
+
+- `EnvTracingConfig` - the overall configuration of traced components of Helidon
```

Comment:
Would it make sense to simply call this `TracingConfig`? The `Env` prefix is a little confusing.

Reply:
Oh, I see you are also introducing a `TracingConfiguration` class in `webserver`. And it looks like you've used the terms "global", "overall" and "application" elsewhere to refer to the `EnvTracingConfig`. Hmm. Lots of terms here to disentangle. Maybe:

1. Rename `TracingConfiguration` to `WebTracingConfig` (using the shorter suffix as used elsewhere).
2. Rename `EnvTracingConfig` to `TracingConfig` (or `AppTracingConfig`).

Reply:
`TracingConfiguration` is already used to configure tracing in gRPC server, so I wanted to be consistent with that. Nevertheless the names are not ideal for the other classes.

Reply:
We could also rename the gRPC class, e.g. `GrpcTracingConfig` (or the full suffix if we want to do that everywhere).

Reply:
I have created a new class `GrpcTracingConfig` and kept the old one for backward compatibility marked as `Deprecated`